make/test.mk

```diff
@@ -1,4 +1,18 @@
+COV_DIR = coverage
+GOPKGS = $(shell go list ./... | grep -v /test)
+
 .PHONY: test
 ## runs all tests with bundles assets
 test: generate
-	go test -count=1 ./...
+	# test the test package seperately, no coverage report
+	go test -count=1 ./test
+	# test the rest of the code, with coverage report	
+	@-mkdir -p $(COV_DIR)
+	@-rm -f $(COV_DIR)/coverage.txt
+	@-for d in $(GOPKGS) ; do \
+	  go test -coverprofile=profile.out -covermode=atomic -coverpkg=$$GOPKGS $$d ; \
```

Comment:
You don't have to test each package individually when using coverpkg. Just run go test once with that flag.

Reply:
Fixed 4b40afb.